src/status_im/ui/screens/events.cljs

```diff
@@ -264,3 +266,11 @@
   :update-geolocation
   (fn [db [_ geolocation]]
     (assoc db :geolocation geolocation)))
+
+;; TODO(oskarth): Put this token in DB
+(register-handler-fx
+ :get-fcm-token
+ (fn [_ _]
+   (notifications/get-fcm-token)
+   {}))
```

Comment:
hm
why does it return `{}`?

Reply:
@alwx just placeholder for effects to come